### ISSUE TITLE
Updating detector from maskrcnn to detectron

### ISSUE THREAD

**Issue by DavidHuji** [microsoft/scene_graph_benchmark#78]
Hi, 
First, thanks for the great repo!
I have a competability issue with the gcc environment that is in my gpu server.
In order to support compatibility with new versions of a few packages (or gcc), I am thinking to change the detector implementation from maskrcnn to [detectron2](https://github.com/facebookresearch/detectron2) or 
[mmdetection](https://github.com/open-mmlab/mmdetection). 
Does anyone encounter a similiar problem? Is there a simpler solution for that? (I cannot use docker and I cannot change the compiler version).
Thanks!

**Comment by dinhanhx** [comment]
I agree with you @DavidHuji . The model should be rescued first is VinVL one

**Comment by Ceazy-Gentleman** [comment]
I agree with you @DavidHuji . The Project is not friendly with 30s GPU!
